Ark.Tools.FtpClient.ArxOne/FtpClientPoolArxOne.cs

```diff
@@ -13,30 +13,31 @@
 
 namespace Ark.Tools.FtpClient
 {
-
     public class FtpClientPoolArxOne : FtpClientBase, IFtpClientPool
     {
         private readonly ArxOne.Ftp.FtpClient _client;
         private readonly SemaphoreSlim _semaphore;
+        private static FtpConfig _ftpConfig = new FtpConfig();
```

Comment:
nope, this is for sure wrong. a static config is wrong in so many way.
means the whole implant is not consistent as each pool has to have its own config.